### DISCUSSION TITLE
What unblocker exploits work on version 115 and how to do all of them?

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#722] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can you guys list all of them and how to do them?

**Comment by rwg100** [comment, marked as answer]
Most effective way:

**STEPS HERE!**
1) Log onto your school chromebook (this should be pretty easy)
2) Open your browser, this is usually something called Chrome and its logo is a circle with a blue circle in the middle surrounded by 3 colors.
3) Open this link: https://github.com/3kh0/ext-remover
4) (Hard part, Only 1% get past this step) Scroll down until you see some image of a cow with blue outlining, you should see text with arrows next to it
5) **(EXTREMELY HARD!!!)** Use an item called "Your Brain" and use another item called "Common sense". You might not have it, because 90% of the population doesn't spawn in with it.
6) Click those arrows so you see text, make sure the text includes the saying: **works on version 115**
7) Do the ones that work, skip the ones that don't.
8) Congrats!

I hope this helped you in your journey to breaking the law and getting banned from all school technology for life. Have a good one!

**Reply by Mazedotexe** [reply]
damn bro got destroyed... didnt even respond lol

**Reply by Vhirttuegaming** [reply]
If accessing this site can result in being banned from school chromebooks, may I ask you why your here?

**Reply by Vhirttuegaming** [reply]
and "destroyed" is a really vague word... can you elaborate on that?

**Reply by rwg100** [reply]
cos its funny

**Reply by funkymokey** [reply]
this guy will never not be funny

**Reply by Su4o** [reply]
"never not" is crazy.

**Reply by ggg23131** [reply]
lo
